communication/contributor-comms/storytelling-resources/sig-spotlight-blog-process.md

```diff
@@ -1,10 +1,27 @@
-3) Interview those people - usually the SIG chairs. We generally recommend doing the interview asynchronously via a Google Doc because written responses are easier to work with. The SIG-Multicluster one was done with a recorded zoom interview but it took a long time to complete.
-4) Write a blog post. The post should be in markdown format and we like to do a round of review on a rough draft before creating a PR. We recommend either hackmd.io or Google Docs for this. Google Docs can be converted to markdown fairly easily.
+1. Approach the SIG in their general Slack channel, presenting yourself and asking for their help
+    * something along the lines of _"Hello, I'm working in contribex-comms in our SIG Spotlight series and would love to write a blog spotlighting your SIG. It's not time-consuming and will be published, providing added visibility and focus on things important for the SIG. Who should I interview?"_ is perfectly fine, but feel free to work around the main idea and add your own words.
+    * You can mention (@) the co-chairs of the SIG in your initial message, or in an additional reply to the thread in case there is no feedback. 
+2. If prompted for more details or as a reply to someone asking "what's next", provide some examples of published SIG Spotlight articles, and describe the process in simple terms.
+    * A summary of the process could be _"I'll write some initial questions and share them with you; based on your answers and eventual suggestions of additional topics to cover we will finish the interview, and from then on we will take care of the publishing process"_.
+4. The interview process can be done in different ways, the following are some suggestions:
+    * Create a Google Docs file with some initial questions, and ask them to write answers, and suggest additional topics if needed; we generally recommend doing the interview asynchronously via a Google Doc because written responses are easier to work with. The SIG-Multicluster one was done with a recorded zoom interview but it took a long time to complete.
+    * We have [example topics](example-topics.md) available for you to use, plus the existing articles that serve as a reference.
+    * You can either write some initial questions, get answers, and stop there, or based on the answers add new ones.
+    * Consider that you're not bound by your initial questions: it's very common for an answer to cover things that would be even better as a reply to a question that isn't there. You can add that questions and refactor answers to improve the flow, as long as the final version is reviewed and approved by the SIG person being interviewed.
+5. With that final version, create a markdown file in hackmd.io, based on the blog markdown template, and share it with the Contribex Comms Blogging lead and the community (the `#contribex-comms` channel).
```

Comment:
#sig-contribex-comms is the channel name.

Reply:
Good catch, fixed!